processccd.rst

```diff
@@ -1,131 +1,100 @@
 
-2/4
 
-ProcessCCD
-================
+##############
+ProcessCcdTask
+##############
 
+ProcessCcd (which is a ``Cmd Line Task``, and thus can be run directly
+by typing processCcd.py at a shell prompt) executes the steps of how
+an image is processed from raw data to a science-grade images and catalogs that can
+be used in analyses.
 
-- `Doxygen link`_
-.. _Doxygen link: https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/classlsst_1_1pipe_1_1tasks_1_1process_ccd_1_1_process_ccd_task.html#ProcessCcdTask_
+In more detail: ProcessCcd as a whole executes many of the functions
+that are in multiple packages in other astronomy analysis frameworks.
 
+- The initial step is to do ``Instrument Signature Removal`` through running :doc:`isrTask <isrtask>` to correct the images for all the issues involved in taking a raw image CCD through to a processed one (e.g. doing the bias and dark current corrections, flat-fielding, etc.).
 
-Next, we will look at the actual steps of how an image is processed
-from raw data to a science-grade image that can be used in analyses.
+- The second step is to do ``Image Characterization`` by running :doc:`characterizeImageTask <charimg>`), which includes for our purposes: object detection, repairing of cosmic ray defects, measuring and subtracting of sky background, and then finally measuring bright sources and using this to estimate background and PSF of an exposure.
+  
+- The last step is doing ``Image Calibration`` by running :doc:`calibrateTask <calibimg>`, which measures faint sources, does the astrometry by fitting an improved WCS to the image, and figures out the photometric zero-point for the image.
 
-Let’s do this by looking inside the primary function which does this
-in the stack -- processCcd.py, which can be executed as so::
+ProcessCcdTask is implemented in the ``lsst.pipe.tasks`` module.
 
-  processCcd.py pathTo/DATA --VariousFlags
 
-(Writing the cmd as ' processCcd.py' directly on the cmd line works
-because processCcd.py is actually an executable which will be in your
-path when it has been set up correctly, which happens as part of the
-DM Stack install.)
-	
-Real example version from the ci_hsc dir to be tried::
+See also
+=========
 
-  processCcd.py $LSSTSW/ci_hsc/DATA --rerun ci_hsc --id visit=903986 ccd=23 --doraise
+(Anything to put here?)
 
-This uses a ‘dataId’, which is a unique identifier for a specific data
-input. This can come in different formats, the one shown here (after the '--id' flag) uses the
-‘visit’ , ‘ccd’ identifiers used to refer to a specific CCD in a
-specific exposure (called a ‘visit’).
+Configuration
+=============
 
-The term 'rerun' originated in SDSS. It simply refers to a single
-processing run, performed with a specified version of the reduction
-code, and with a specific set of configuration parameters. The
-assumption is that within a given rerun, the data have been handled
-in a homogeneous way.
+Flags  and utility variables
```

Comment:
We need a single term for these instead of "Flags and utility variables."

I vote for "Parameters." Matches numpydoc lingo and it's fundamentally the same idea.